lib/Compile.js

```diff
@@ -168,56 +188,172 @@ var Compile = module.exports = function (options) {
             var styles;
             var opacity;
             var path;
+            
+            //var printerCutLineGroup = document.createElement("g");
+            
+            var printerCutline = document.createElement("polygon");
+            var cutlinePoints = "";
+            
+            if(options.stickerSheet) {
+              printerCutline.setAttribute("style", "fill:none;stroke:#FF00FF;stroke-width:1");
+            }
             Geometry.all.forEach(function(obj, key) {
-                  
-                  styles = '';
-                  path  = document.createElement("path");
-                  
-                  // Theres an issue with Y being flipped, maybe to do with viewBox so * -1 seems to work fancy like!
-                  // Create one trixel
-                  path.setAttribute('d', 'M ' + obj.v0.x * unitScale + ' '+ (obj.v0.y * unitScale)*-1 +'  L ' + obj.v1.x * unitScale + ' '+ (obj.v1.y * unitScale)*-1 +' L ' + obj.v2.x * unitScale + ' '+ (obj.v2.y * unitScale)*-1 +'z');
-                  
-                  // Pulling the color from the JSON data
-                  if (jsonData[key]) {
-                    
-                      if (typeof jsonData[key].color !== 'undefined') {
-                        styles += 'fill:'+jsonData[key].color+';';
-                      }
-                      
-                      if (typeof jsonData[key].opacity !== 'undefined' && jsonData[key].opacity != 1) {
-                        styles += 'fill-opacity:'+jsonData[key].color+';';
-                      }
-                      
-                      path.setAttribute('style',  styles);
-                  } else {
-                    
-                      path.setAttribute('style',  'fill:'+randomRGB()+';');
+              
+              if(options.stickerSheet) {
```

Comment:
Without checking my source at the moment I would say correct-o-tron! There are some deallys with the edges and how it all relates to the grid but yes. However the larger question is, what are you trying to do? I saw another email that was discussing a hard value of 139 if double that I see 278 which is a common number to be. 
Are you trying to get the pointing vertice outward? If so that not the right way.

for List.length
List[i].vertice[0] // its either 0 1 or 2 all of them
// move the 0,1 or 2 index outward all of them will follow this flow.

Its all built on flow.